app/resources/v1/pacbio/runs/well_resource.rb

```diff
@@ -3,10 +3,62 @@
 module V1
   module Pacbio
     module Runs
-      # WellResource
+      # @todo This documentation does not yet include a detailed description of what this resource represents.
+      # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+      # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+      #
+      # @note Access this resource via the `/api/v1/pacbio/runs/well` endpoint.
+      #
+      # Provides a JSON:API representation of {Pacbio::Well}.
+      #
+      # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
+      # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package
+      # for the service implementation of the JSON:API standard.
       class WellResource < JSONAPI::Resource
         model_name 'Pacbio::Well'
 
+        # @!attribute [rw] row
```

Comment:
The only thing about having this here is that the documentation will become stale if there are changes. This is why I move it come from the configuration.

Reply:
I agree. I think we probably could use [`@macro` ](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#:~:text=it%20is%20ignored.-,Macros,-Fortunately%20YARD%200.7.0) for a certain extent in case we want to get the variable name out from the comment listed here. I will try to find if there's a way to move it a configurable level.